lib/ffmpeg/encoderbase.py

```diff
@@ -238,15 +238,17 @@ def check_bitrate(self):
     get_media()._set_test_details(
       size_encoded = encsize,
       bitrate_actual = "{:-.2f}".format(bitrate_actual))
+    if "bitrate_gap" in vars(self).get("skip_check", []):
+      return
 
-    if "cbr" == self.rcmode:
+    if "cbr" == self.rcmode and vars(self).get("skip_check", []):
```

Comment:
This change is not needed in the conditionals anymore.  You've already checked it above.

Reply:
But, after I tried to delete this change, the case still has a bitrate_gap issues

Reply:
I can't see what you're actually doing that leads you to that conclusion.  But you're doing something wrong because if I fix it locally, it works for me.